docs/usage.md

```diff
@@ -62,6 +62,57 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Handling git branches
+
+Use `git-branch` command to work on branch related cases on base repo and any enabled modules.
+By default, a git checkout action will be performed to the revision
+in the base repository. However, the `git-branch --operation` option can be provided to change this
+behavior.
+
+The option takes one of the following as an argument:
+
+* **checkout** [default] - Perform a `git checkout` to checkout the found revision.
```

Comment:
All these options say they use the "found" revision. What is meant by "found" here?

Reply:
Removed